torch/linalg/__init__.py

```diff
@@ -813,6 +813,102 @@
     https://www.netlib.org/lapack/lug/node128.html
 """)
 
+ldl_factor_ex = _add_docstr(_linalg.linalg_ldl_factor_ex, r"""
+linalg.ldl_factor_ex(A, *, upper=False, hermitian=False, check_errors=False, out=None) -> (Tensor, Tensor, Tensor)
+
+Computes the LDL decomposition of a Hermitian/symmetric indefinite matrix.
+When :attr:`A` is complex valued it can be Hermitian (:attr:`hermitian` = `True`)
+or symmetric (:attr:`hermitian` = `False`).
```

Comment:
The first paragraph should be just one line for it to be formatted properly in the linear algebra docs page.